integrate-bamboo-github.md

```diff
@@ -0,0 +1,33 @@
+# Bamboo Integration
+
+## 1. Setup Bamboo
+
+### Installation
+
+Download Bamboo from https://www.atlassian.com/software/bamboo. Follow their [installation guide](https://confluence.atlassian.com/bamboo/getting-started-with-bamboo-289277283.html?_ga=2.95849887.246880307.1531709232-1995250601.1528082340). 
+
+### Create a project and link to GitHub
```

Comment:
I still don't think this is necessary as we already assumed the readers know how to use this feature.

Reply:
Linking to github happens naturally when you create a project so i think it's okay cause i should show them basics of creating a new project anyway